modules/installation-guide/pages/pulling-images-used-by-che-to-a-private-registry.adoc

```diff
@@ -0,0 +1,7 @@
+[id="pulling-images-used-by-che-to-a-private-registry"]
+// = Pulling images used by Che to a private registry
+:navtitle: Pulling images used by Che to a private registry
+:keywords: installation-guide, pulling-images-used-by-che-to-a-private-registry
+:page-aliases: .:pulling-images-used-by-che-to-a-private-registry
```

Comment:
Why is this alias here? Is the current ID not the same as the alias?
```suggestion
```

Reply:
I don't know. I asked Fabrice for a review, so it passed through him or he added it there for some reason. This is my first work since antora, so I am still a little bit in exploring how to do stuff.